### PR TITLE
Installer release notes for 4.8

### PR DESCRIPTION
**Pull request by codyhoag**
Preview: https://deploy-preview-32054--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes.html#ocp-4-8-installation-and-upgrade

### CODE REVIEW

**Comment by codyhoag** [comment]
@staebler can you give this a sanity check? I will copy over the BYO hosted zone/IAM role release notes for the 4.7 z-stream release notes when they're released.

@katherinedube do you mind confirming, too?

Thanks!

**Comment by katherinedube** [comment]
lgtm

**Review by staebler — COMMENTED** [review]
LGTM

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -49,6 +49,27 @@ This release adds improvements related to the following components and concepts.
...
+
+You can now define an already existing resource group to install your cluster to on Azure by defining the `platform.azure.resourceGroupName` field in the `install-config.yaml` file. This resource group must be empty and only used for a single cluster; the cluster components assume ownership of all resources in the resource group.
+
+If you limit the service principal scope of the installation program to this resource group, you must ensure all other resources used by the installation program in your environment have the necessary permissions, such as the public DNS zone and virtual network. Destroying the cluster using the installation program deletes the user-defined resource group.
+
+[id="ocp-4-8-installation-existing-iam-roles-aws"]
+==== Using existing IAM roles for clusters on AWS
+
+You can now define a pre-existing Amazon Web Services (AWS) IAM role for your machine instance profiles by setting the `compute.platform.aws.iamRole` and `controlPlane.platform.aws.iamRole` fields in the `install-config.yaml` file. This allows you to match naming schemes and include predefined permissions boundaries for your IAM roles for clusters installed on AWS.
```
Comment (skrthomas) [inline release_notes/ocp-4-8-release-notes.adoc:62]: Suggestion: This allows you to match naming schemes. For clusters installed on AWS, you can include predefined permissions boundaries for your IAM roles.

File: release_notes/ocp-4-8-release-notes.adoc [hunk 2]
```diff
@@ -49,6 +49,27 @@ This release adds improvements related to the following components and concepts.
...
+
+[id="ocp-4-8-installation-pre-existing-hosted-zones-aws"]
+==== Using pre-existing Route53 hosted private zones on AWS
+
+You can now define an existing Route 53 private hosted zone for your cluster by setting the `platform.aws.hostedZone` field in the `install-config.yaml` file. You can only use a pre-existing hosted zone when also supplying your own VPC.
+
+[id="ocp-4-8-installation-increase-gcp-subnets-within-machine-cidr"]
+==== Increasing the size of GCP subnets within the machine CIDR
+
+The {product-title} installation program for Google Cloud Platform (GCP) now creates subnets as large as possible within the machine CIDR. This allows the cluster to use a machine CIDR appropriately sized to accommodate the number of nodes in the cluster.
```
Comment (skrthomas) [inline release_notes/ocp-4-8-release-notes.adoc:72]: This allows the cluster to use a**n appropriately sized** machine CIDR to accommodate the number of nodes in the cluster.

**Review by skrthomas — APPROVED** [review]
A comment and suggestion, otherwise, LGTM!

File: release_notes/ocp-4-8-release-notes.adoc (hunk 2 above)
Comment (skrthomas) [reply release_notes/ocp-4-8-release-notes.adoc:72]: Should say, I'm unsure about this comment as it pertains to IBM Style guide usage of adverbs. Maybe best to ignore it :)

**Pull request merged**